src/web/js/check-ui.js

```diff
@@ -189,8 +189,9 @@
 
         FailingTestSkeleton.prototype.refresh = function refresh() {
           var snippets = this.tombstone.querySelectorAll(".CodeMirror");
-          for (var i = 0; i < snippets.length; i++) {
-            snippets[i].CodeMirror.refresh();
+          for (let i = 0; i < snippets.length; i++) {
+            window.requestAnimationFrame(
```

Comment:
> Would it be better to hoist the request out of the loop, and refresh all the cm instances at once?

Jank is what happens when the browser is too caught up running javascript to repaint effectively. If my understanding of `requestAnimationFrame` is correct, using multiple calls of `requestAnimationFrame` leaves the decision of how refreshes and repaints are interleaved up to the browser. If `refresh` is very quick, the browser may be able to churn through several `requestAnimationFrame` before it paints the frame. Using a monolithic call of `requestAnimationFrame` runs the risk of having an extremely long frame.

On the other hand, since these are only snippets being refreshed here, it's probably safe to assume that the `refresh` calls are very quick. I haven't profiled enough to say for sure.